README.md

```diff
@@ -33,7 +33,19 @@ cd ckeditor5-build-strapi-wysiwyg
 npm i
 ```
 
-3. Open **demo/index.html** in your system browser.
+3. Run the project on a server
+
+```
+python -m SimpleHTTPServer 8080
+```
+
+or
+
+```
+http-server -p 8080
+```
+
+4. Open **demo/index.html** in your system browser through http://localhost:8080/demo/.
```

Comment:
Why do you need web server for demo? Did you just try to open **demo/index.html** in browser? It works for me.

Reply:
The demo page loads, but the editor doesn't with the following console error: `There was a problem initializing the editor. Cross site scripting is only allowed for HTTP.`